browser/_templates/react-boilerplate/new/index.ejs.t

```diff
@@ -0,0 +1,19 @@
+---
+to: src/Index.js
+---
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import {
+  BrowserRouter as Router
+} from "react-router-dom"
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(<Router><App /></Router>, document.getElementById('root'));
```

Comment:
Setting up React Router should be a separate PR

Reply:
Oh wait - I see what you're doing!

We're going to setup index, app, etc in a later PR - we only need a generator for repeated actions (like creating components) - good thinking though!

Reply:
Oh wow, this is embarrassing, the entire react-boilerplate was accidentally included from some other project I was working on, totally didn't even see that I left it in the directory!